language/python/run-containers.md

```diff
@@ -19,20 +19,20 @@ A container is a normal operating system process except that this process is iso
 To run an image inside of a container, we use the `docker run` command. The `docker run` command requires one parameter which is the name of the image. Let’s start our image and make sure it is running correctly. Run the following command in your terminal.
 
 ```shell
-$ docker run python-docker
+docker run python-docker
 ```
 
 After running this command, you’ll notice that you were not returned to the command prompt. This is because our application is a REST server and runs in a loop waiting for incoming requests without returning control back to the OS until we stop the container.
 
 Let’s make a `GET` request to the server using the `curl` command.
 
 ```shell
-$ curl --request POST \
-  --url http://localhost:8000/test \
-  --header 'content-type: application/json' \
-  --data '{
-	"msg": "testing"
-}'
+curl http://localhost:8000/
```

Comment:
When looking at the tutorial I thought we should just go with port 5000 everywhere.